lib/recurly/purchase.php

```diff
@@ -2,19 +2,20 @@
 /**
  * Class Recurly_Purchase
  * @property Recurly_Account $account The account for the purchase. Can create an account or use existing.
- * @property Recurly_Adjustment[] $adjustments The array of adjustments for the purchase
- * @property string $collection_method The invoice collection method ('automatic' or 'manual')
- * @property string $currency The currency to use in this invoice
- * @property string $po_number The po number for the invoice
- * @property integer $net_terms The net terms of the invoice
- * @property string[] $coupon_codes An array of coupon codes to apply to the purchase
- * @property Recurly_Subscription[] $subscriptions An array of subscriptions to apply to the purchase
- * @property Recurly_GiftCard $gift_card A gift card to apply to the purchase
+ * @property Recurly_Adjustment[] $adjustments The array of adjustments for the purchase.
+ * @property string $collection_method The invoice collection method ('automatic' or 'manual').
+ * @property string $currency The currency to use in this invoice.
+ * @property string $po_number The po number for the invoice.
+ * @property integer $net_terms The net terms of the invoice.
+ * @property string[] $coupon_codes An array of coupon codes to apply to the purchase.
+ * @property Recurly_Subscription[] $subscriptions An array of subscriptions to apply to the purchase.
+ * @property Recurly_GiftCard $gift_card A gift card to apply to the purchase.
```

Comment:
All of the rest of the `@property` declarations end with a period, so I decided to fix this while I was in here